Mathlib/GroupTheory/OrderOfElement.lean

```diff
@@ -962,6 +963,22 @@ theorem pow_gcd_card_eq_one_iff : x ^ n = 1 ↔ x ^ gcd n (Fintype.card G) = 1 :
     let ⟨m, hm⟩ := gcd_dvd_left n (Fintype.card G)
     rw [hm, pow_mul, h, one_pow]⟩
 
+lemma smul_eq_of_le_smul
+    {G : Type*} [Group G] [Finite G] {α : Type*} [PartialOrder α] {g : G} {a : α}
+    [MulAction G α] [CovariantClass G α HSMul.hSMul LE.le] (h : a ≤ g • a) : g • a = a := by
```

Comment:
there i now an abbrev for this
```suggestion
    [MulAction G α] [SMulLeftMono G α] (h : a ≤ g • a) : g • a = a := by
```
or something

Reply:
We have `PosSMulMono`, but I can't seem to find `SMulMono`.